composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "asgorobets/soap-extension",
+  "name": "behat/soap-extension",
```

Comment:
Definitely, I was thinking this can't be different from repository name on Github, but I was wrong =)